index.js

```diff
@@ -676,6 +677,9 @@ Peer.prototype._onTrack = function (event) {
   var self = this
   if (self.destroyed) return
   self._debug('on track')
+  var id = event.streams[0].id
+  if (self._previousStreams.indexOf(id) !== -1) return // Only fire one 'stream' event per track
```

Comment:
I think this comment could be clearer. How about "Only fire one 'stream' event, even though there may be multiple tracks per stream"